tools/autograd/derivatives.yaml

```diff
@@ -1313,8 +1313,8 @@
   grad_output: grad.to(output.dtype()) - (grad.to(output.dtype()) * output.exp()).sum(dim, true)
   self: log_softmax_double_backward(grad.to(output.dtype()), grad_output, dim, output).to(self.dtype())
 
-- name: leaky_relu_backward(Tensor grad_output, Tensor self, Scalar negative_slope) -> Tensor
-  grad_output: leaky_relu_backward(grad, self, negative_slope)
+- name: leaky_relu_backward(Tensor grad_output, Tensor self, Scalar negative_slope, bool inplace=False) -> Tensor
+  grad_output: leaky_relu_backward(grad, self, negative_slope, inplace)
```

Comment:
This leaky relu backward actually corresponds to an out of place leaky relu.
Even if the forward was inplace, the backward never modifies its input inplace, so the backward is an out of place leaky relu.

Reply:
@glaringlee "inplace" for `leaky_relu_backward` just means "was the forward an in-place function?", right? If so we should rename the argument to `leaky_relu_backward` as `was_forward_inplace` or something

Reply:
Yes, we discussed it offline.
The second argument of `leaky_relu_backward` should be renamed from `self` to `self_or_result` and the flag should tell if it is one or the other.